velox/exec/Window.cpp

```diff
@@ -574,8 +613,16 @@ vector_size_t Window::callApplyLoop(
   // This function requires that the currentPartition_ is available for output.
   VELOX_DCHECK_NOT_NULL(currentPartition_);
   while (numOutputRowsLeft > 0) {
-    auto rowsForCurrentPartition =
-        currentPartition_->numRows() - partitionOffset_;
+    auto totalRows = currentPartition_->numRows();
+    auto rowsForCurrentPartition = totalRows - partitionOffset_;
```

Comment:
Don't understand when rowsForCurrentParititon < 0. Please can you explain

Reply:
@aditi-pandit Now we can use partial() method to check whether the partition is partial.